server/src/main/java/de/pnp/manager/ui/network/SpawnInventoryView.java

```diff
@@ -0,0 +1,48 @@
+package de.pnp.manager.ui.network;
+
+import de.pnp.manager.main.LanguageUtility;
+import de.pnp.manager.model.character.Inventory;
+import de.pnp.manager.model.manager.InventoryHandler;
+import de.pnp.manager.ui.View;
+import de.pnp.manager.ui.ViewFactory;
+import de.pnp.manager.ui.part.NumberField;
+import javafx.beans.property.*;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.VBox;
+
+public class SpawnInventoryView extends View {
```

Comment:
java convention: classes should not be named with verbs at front
InventorySpawningView